package.json

```diff
@@ -304,7 +304,7 @@
     "mapbox-gl": "^1.12.0",
     "markdown-it": "^12.3.2",
     "markdown-it-link-attributes": "^4.0.0",
-    "moment": "~2.24.0",
+    "moment": "^2.29.2",
```

Comment:
Any potential side effects from this upgrade? If so I'd recommend doing it in a separate PR

Reply:
Same. unless this is require, let's move it to a new PR